internal/pkg/warewulfd/api/api.go

```diff
@@ -71,6 +71,7 @@ func Handler(auth *config.Authentication, allowedNets []net.IPNet) *web.Service
 			r.Method(http.MethodGet, "/{name}/file", nethttp.NewHandler(getOverlayFile()))
 			r.Method(http.MethodPut, "/{name}", nethttp.NewHandler(createOverlay()))
 			r.Method(http.MethodDelete, "/{name}", nethttp.NewHandler(deleteOverlay()))
+			r.Method(http.MethodPost, "/{name}/build", nethttp.NewHandler(buildOverlay()))
```

Comment:
@JasonYangShadow I think this should be on `/api/nodes/overlays/build?overlays=`; that is, it's building "all" node overlays (an endpoint already present) but filtering it for nodes that are using a given overlay.

Reply:
thanks! updated as requested